components/PersonView/PersonDetails.jsx

```diff
@@ -10,68 +11,99 @@ const PersonDetails = ({
   gender,
   addressList,
   phoneNumber,
-}) => (
-  <>
-    <h1 className="govuk-fieldset__legend--l gov-weight-lighter">
-      {firstName} {lastName}
-    </h1>
-    <h2 className="govuk-fieldset__legend--m govuk-custom-text-color">
-      PERSON DETAILS
-    </h2>
-    <hr className="govuk-divider" />
-    <dl className="govuk-summary-list">
-      <div className="govuk-summary-list__row">
-        <dt className="govuk-summary-list__key">Mosaic ID:</dt>
-        <dd className="govuk-summary-list__value">#{mosaicId}</dd>
-      </div>
-      <div className="govuk-summary-list__row">
-        <dt className="govuk-summary-list__key">Date of birth</dt>
-        <dd className="govuk-summary-list__value">
-          {new Date(dateOfBirth).toLocaleDateString('en-GB')}
-        </dd>
-      </div>
-      <div className="govuk-summary-list__row">
-        <dt className="govuk-summary-list__key">NHS number</dt>
-        <dd className="govuk-summary-list__value">{nhsNumber}</dd>
+  expandView = false,
+  nameSize = 'l',
+}) => {
+  const [expandDetails, setExpandDetails] = useState(false);
+  const handleToggle = () => {
+    !expandDetails ? setExpandDetails(true) : setExpandDetails(false);
+  };
+  return (
+    <>
+      <div className={expandView ? 'govuk-expand-details' : 'govuk-detail'}>
+        <h1
+          className={`govuk-fieldset__legend--${nameSize} gov-weight-lighter govuk-expand-title`}
+        >
+          {firstName} {lastName}
+        </h1>
+        {expandView && !expandDetails ? (
+          <span className="govuk-expand-toggle" onClick={handleToggle}>
+            Expand view
+          </span>
+        ) : null}
+        {expandView && expandDetails ? (
+          <span className="govuk-expand-toggle" onClick={handleToggle}>
+            Collapse view
+          </span>
+        ) : null}
```

Comment:
We can simplify this:

```suggestion
        {expandView && (
          <span
            className="govuk-expand-toggle"
            onClick={() => setExpandDetails(!expandDetails)}
          >
            {expandDetails ? 'Expand' : 'Collapse'} view
          </span>
        )}
```